sphinx_gallery/gen_gallery.py

```diff
@@ -688,6 +703,232 @@ def write_computation_times(gallery_conf, target_dir, costs):
             fid.write(hline)
 
 
+def write_api_entries(app, what, name, obj, options, lines):
+    if 'api_entries' not in app.config.sphinx_gallery_conf:
+        app.config.sphinx_gallery_conf['api_entries'] = \
+            {entry_type: set() for entry_type in
+             ('class', 'method', 'function', 'module',
+              'property', 'attribute')}
+    app.config.sphinx_gallery_conf['api_entries'][what].add(name)
+
+
+def init_api_usage(gallery_dir):
+    with codecs.open(os.path.join(gallery_dir, 'sg_api_usage.rst'), 'w',
+                     encoding='utf-8'):
+        pass
+
+
+def _make_graph(fname, entries, gallery_conf):
+    """Make a graph of unused and used API entries.
+
+    The used API entries themselves are documented in the list, so
+    for the graph, we'll focus on the number of unused API entries
+    per modules. Modules with lots of unused entries will be colored
+    red, those with no unused entries will be colored green and
+    modules with intermediate unused entries will be colored yellow.
```

Comment:
Looks good, just this nitpick, from the code:
```python
                    if leaf_count > 10:
                        color = 'red'
                    elif leaf_count > 5:
                        color = 'orange'
                    else:
                        color = 'yellow'
```
it looks like the colours are different than described above?

Reply:
Ah, I just missed orange.

Reply:
Thanks!

Reply:
I might have missed it but i didn't see anything being green?

Reply:
Is it yellow is anything =<5?

Reply:
Yes yellow is less than 5. Shoot they're actually blue for no unused, that's the default color.